ref/feature/socialLogin/socialLogin copy.examples

```diff
@@ -0,0 +1,89 @@
+== About this feature
```

Comment:
Is this an accidental checkin?

Reply:
Yeah :( sorry, I thought I was being careful not to do that but it clearly slipped through.